src/web.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add some comments to clarify the need of web.config

Reply:
The original commit mentioned the reason as "Add web.config to prevent 404 via IIS".
Ref : https://github.com/CHBase/smart-ng-fhir-app/pull/1/commits/686afc7b7043ecd374968a228f4be8d1a97e996e

But I do think it might be better to insert some context for IIS setup in README.md

Reply:
FYI, @jeffy-grh  I have added details regarding IIS setup in https://github.com/CHBase/smart-ng-fhir-app/pull/1/commits/96737200b67db68a7e396ec295ac0f72067e7c34